lib/order.rb

```diff
@@ -0,0 +1,23 @@
+class Order
+
+  attr_reader :list, :total
+
+  def initialize
+    @list = []
+    @total = total
+  end
+
```

Comment:
`#add` is a bit hard to read.  Can you find a way of making this method simpler?

Perhaps you could abstract some of the behaviour into another method/s.  For example, it's useful to abstract complex conditions so that they can be named.

```
def add(dish, amount)
  return @list << {name: dish[:name], price: dish[:price], amount: amount} if not_on_list?(item)
  <blah blah blah>
end

def not_on_list?(item)
  @list.select{|item| item[:name] == dish[:name]}.empty?
end
```

OR, using `#include?`

```
def not_on_list?(item)
  @list.map(&:name).include?( item[:name] )
end
```

How does `#include?` compare to `#select` + `#empty?` in terms of communicating intent?